lib/ansible/module_utils/aci.py

```diff
@@ -66,6 +66,7 @@
     aep=dict(aci_class='infraAttEntityP', mo='infra/attentp-', key='name'),
     ap=dict(aci_class='fvAp', mo='ap-', key='name'),
     bd=dict(aci_class='fvBD', mo='BD-', key='name'),
+    bd_dhcp_label=dict(aci_class='dhcpLbl', mo='dhcplbl-', key='name'),
```

Comment:
please remove changes to aci.py and pull down the latest. We have refactored this to accommodate some special cases in the URLs, and provided a cleaner interface for the modules.